.github/workflows/setup/apt-packages.txt

```diff
@@ -0,0 +1,40 @@
+alien
+capnproto
+ffmpeg
+gcc-arm-none-eabi
+gnome-screenshot
+libavcodec-dev
+libavdevice-dev
+libavfilter-dev
+libavformat-dev
+libavutil-dev
+libbz2-dev
+libcapnp-dev
+libcurl4-openssl-dev
+libeigen3-dev
+libglew-dev
+libgles2-mesa-dev
+libglfw3-dev
+liblzma-dev
```

Comment:
We can't have 2 sources of truth for these dependencies. They are currently defined in `tools/install_ubuntu_dependencies.sh`